corehq/doctypemigrations/README.md

```diff
@@ -228,3 +249,8 @@ Keep in mind that this will likely incur some reindexing overhead in the source
 
 If you run `$ ./manage.py run_doctype_migration user_db_migration --stats`, you
 should see that only the target database has these doc types in it now.
+
+
+### Stop syncing irrelevant views to the old db
+If you synced any `couchapps` views to both dbs for the migration, you should
+register those only to the new db (in `corehq/couchapps/__init__.py`).
```

Comment:
I'd maybe reword this. My convention was to sync to both `APPS_DB` and `NEW_APPS_DB`—before the migration they're different, after the migration they're the same. Thus no cleanup step is necessary except when down the road you remove the reference to NEW_APPS_DB entirely. (We've never done this step so far because I think it would screw up the blocking mirgrations, etc., and it's easy to keep around for a while before it becomes a pain.)